vita3k/modules/module_parent.cpp

```diff
@@ -220,7 +221,10 @@ SceUID load_module(EmuEnvState &emuenv, const std::string &module_path) {
         LOG_ERROR("Failed to read module file {}", module_path);
         return SCE_ERROR_ERRNO_ENOENT;
     }
-    SceUID module_id = load_self(emuenv.kernel, emuenv.mem, module_buffer.data(), module_path, emuenv.log_path);
+
+    const std::vector<Patch> patches = get_patches(emuenv.patch_path, emuenv.io.title_id);
```

Comment:
You are re-loading the patches from their file every time a module is being loaded. You should at least cache them or only load them when loading the eboot module considering it's the only one which can be patched right now.